packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -251,3 +253,43 @@ export const getOptionPerUnit = (unit: string, show_tick_from_5: boolean): { val
 
     return [[]];
 };
+
+export const getProposalRequestObject = ({
+    is_take_profit_input,
+    is_enabled,
+    should_set_validation_params,
+    should_subscribe = false,
+    trade_store,
+    trade_type,
+    new_input_value,
+}: {
+    is_take_profit_input: boolean;
+    is_enabled: boolean;
+    should_set_validation_params: boolean;
+    should_subscribe?: boolean;
+    trade_store: TTradeStore;
+    trade_type: string;
+    new_input_value?: string;
+}) => {
+    /* In order to get validation params for Multipliers when TP and SL are empty, 
+            we send '1' first, get validation params and set them into the state.*/
+    const input_value = should_set_validation_params ? '1' : new_input_value;
+    const store = {
+        ...trade_store,
+        ...{
+            ...(is_take_profit_input ? { has_take_profit: is_enabled } : { has_stop_loss: is_enabled }),
+            has_cancellation: false,
+            ...(is_take_profit_input
+                ? { take_profit: is_enabled ? input_value : '' }
+                : { stop_loss: is_enabled ? input_value : '' }),
+        },
+    };
+    const request = createProposalRequestForContract(
+        store as Parameters<typeof createProposalRequestForContract>[0],
+        trade_type
+    ) as Omit<ReturnType<typeof createProposalRequestForContract>, 'subscribe'> & { subscribe?: number };
```

Comment:
is this typecast needed? typescript complains if you delete subscribe?

Reply:
Yes, I can delete only something optional, otherwise ts throws error

Reply:
I can replace 'delete' with alternative way, like this:
```
const laptop = {
    brand: "Pineapple",
    model: "New fancy laptop",
    year: 2023,
}

const {model, ...newLaptop} = laptop;
```
`newLaptop` won't contain field `model`
What do you think? Leave type assertion or apply alternative way? of deleting

Reply:
its fine